<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>processing</class>
 <widget class="QDialog" name="processing">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>281</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>281</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Progress</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lIcon">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>100</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
font: bold 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lfondo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color:qlineargradient(spread:pad, x1:0.519, y1:0.977273, x2:0.52, y2:0.128, stop:0.767045 rgba(0, 0, 0, 200), stop:1 rgba(255, 255, 255, 240));
	/*background-color: rgba(0, 0, 0, 200);*/
border: 5px; border-radius:50px;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lLoading">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>18</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
 border-radius:5 px;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lfondo_2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>0</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgba(255, 255, 255, 230);
	/*background-color: rgba(0, 0, 0, 200);*/
border:5px; border-radius:20px;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lfondo_2</zorder>
  <zorder>lfondo</zorder>
  <zorder>lIcon</zorder>
  <zorder>progressBar</zorder>
  <zorder>lLoading</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
